web/ui/react-app/src/pages/starting/Starting.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useEffect } from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { Progress, Alert } from 'reactstrap';
+import Logo from './Logo';
+
+import { useFetchReadyInterval, WalReplayData } from '../../hooks/useFetch';
+import { usePathPrefix } from '../../contexts/PathPrefixContext';
+
+interface StartingContentProps {
+  isResponding: boolean;
+  status?: WalReplayData;
+}
+
+export const StartingContent: FC<StartingContentProps> = ({ status, isResponding }) => {
+  if (!isResponding) {
+    return (
+      <Alert color="danger">
+        <strong>Error:</strong> Server is not responding
+      </Alert>
+    );
+  }
+
+  return (
+    <div className="text-center m-3">
+      <Logo height="400" />
+      <div className="m-4">
```

Comment:
Could you share a screenshot in the PR of what this page looks like when the WAL is loading?

Reply:
Just updated the comment with a (sadly low quality) GIF.